lib/internal/cluster/worker.js

```diff
@@ -3,8 +3,7 @@ const EventEmitter = require('events');
 const internalUtil = require('internal/util');
 const util = require('util');
 const defineProperty = Object.defineProperty;
-const suicideDeprecationMessage =
-    'worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.';
+const deprecationMessage = 'Please use worker.exitedAfterDisconnect instead.';
```

Comment:
It's hard for me to imagine a scenario where it would be repeated over and over in the logs, as the warning is one-time per process.